spec/unit/hanami/events_spec.rb

```diff
@@ -8,23 +8,23 @@
   end
 
   describe '#broadcast' do
+    let(:event_pattern) { 'user.created' }
+
     before do
-      event.subscribe('user.created') { |payload| payload }
+      event.subscribe(event_pattern) { |payload| payload }
```

Comment:
I know that we treat event name as pattern in matcher, but in this case we could use `event_name` as the name of the variable.